src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -50,6 +51,9 @@
     @FXML
     private StackPane resultDisplayPlaceholder;
 
+    @FXML
+    private StackPane deliveryDisplayPlaceholder;
```

Comment:
```suggestion
    private StackPane deliveryItemListPlaceholder;
```

Just to keep it more consistent with itemListPanelPlaceholder, what do you think? Just some minor stuff